### ISSUE TITLE
24/7 script problems

### ISSUE THREAD

**Issue by sebo361** [InstaPy/InstaPy#4207]
Hi guys,

i am currently using the script below and i have following questions:

1) How do i make sure that i am not interacting with my currently accounts that i follow? I used `account_following = session.grab_following(username="XXX", amount="full", live_match=False, store_locally=True)` in combination with `  session.set_ignore_users(account_following)` but the problem is that when the first session ends and a second session starts, the new followed accounts from the first session get ignored too. But i just dont want to interact all the time with my accounts following from the first session. Any idea?

2) When using `session.unfollow_users(amount=random.randint(75,150), InstapyFollowed=(True, "nonfollowers"), style="FIFO", unfollow_after=24*60*60, sleep_delay=501)` I have the feeling that it is not waiting 24 hours for unfollowing accounts but unfollowing straigth after it has followed the accounts. How can i make sure that it unfollows account only after 24 hours? can i split it up in two session and session 2 remembers who was followed in session 1?

3) I got a soft Ban for liking photos. What are the limits for quoata supervisor so no Ban vom instagram occurs?

```
# -*- coding: UTF-8 -*-
from instapy import InstaPy
import schedule
import time
import requests
import random
from random import randint
from datetime import datetime
import os
from tempfile import gettempdir
from selenium.common.exceptions import NoSuchElementException
  
insta_username = 'XXX'
insta_password = 'XXX'
dont_likes = ['XXX']
targets = ['XXX']
target_business_categories = ['XXX']
like_tag_list = ['XXX']

session = InstaPy(username=insta_username,
                  password=insta_password,
                  bypass_suspicious_attempt=False,
                  headless_browser=True,
                  disable_image_load=True,
                  multi_logs=True)

def follow():

  session.login()
  account_following = session.grab_following(username="XXX", amount="full", live_match=False, store_locally=True)
  session.set_relationship_bounds(enabled=True,
      potency_ratio=None,
      delimit_by_numbers=True,
      max_followers=60000,
      max_following=30000,
      min_followers=30,
      min_following=30,
      min_posts=10)

  session.set_quota_supervisor(enabled=True, sleep_after=["likes", "follows", "unfollows", "server_calls_h"], sleepyhead=True, stochastic_flow=True, notify_me=True,
                            peak_likes=(50, 450),
                              peak_follows=(50, 350),
                               peak_unfollows=(10, 100),
                                peak_server_calls=(400, None))

  session.set_skip_users(skip_private=True,
             private_percentage=70,
             skip_no_profile_pic=True,
             no_profile_pic_percentage=90,
             skip_business=True,
             business_percentage=90,
             dont_skip_business_categories=[target_business_categories])

  session.set_ignore_users(account_following)
  session.set_dont_like(dont_likes)
  session.set_delimit_liking(enabled=True, max=300, min=15)
  session.set_user_interact(amount=2, randomize=True, percentage=80,media='Photo')
  session.set_do_follow(enabled=True, percentage=80, times=1)
  session.set_do_like(enabled=True, percentage=90)

  # actions
  session.follow_user_followers(random.sample(targets, 6),
                                  amount=random.randint(30, 50),
                                  randomize=True, sleep_delay=600,
                                  interact=True)
  session.like_by_tags(random.sample(like_tag_list, 3), amount=random.randint(5, 15), interact=True)
  session.follow_by_tags(random.sample(like_tag_list, 3), amount=randint(15,30))
  session.follow_user_followers(random.sample(targets, 4),
                                  amount=random.randint(40, 70),
                                  randomize=True, sleep_delay=600,
                                  interact=True)
session.unfollow_users(amount=random.randint(75,150), InstapyFollowed=(True, "nonfollowers"), style="FIFO", unfollow_after=24*60*60, sleep_delay=501)

def unfollow():
  session.login()
  session.unfollow_users(amount=random.randint(75,150), InstapyFollowed=(True, "nonfollowers"), style="FIFO", unfollow_after=24*60*60, sleep_delay=501)
  
  
  
def xunfollow():
  session.unfollow_users(amount=random.randint(75, 100),
                     InstapyFollowed=(True, "all"), style="FIFO",
                     unfollow_after=168 * 60 * 60, sleep_delay=600)
    

schedule.every(5).hours.do(follow)
schedule.every().day.at("00:05").do(unfollow)
  
schedule.every().sunday.at("03:00").do(xunfollow)

while True:
  schedule.run_pending()
  time.sleep(1)

```
